src/test/features/response/helpers/statementOfMeansFeature.ts

```diff
@@ -65,12 +65,16 @@ describe('StatementOfMeansFeature', () => {
       itShouldBeEnabledForNonBusinessAndDisabledForBusinessDefendants(new ResponseDraft().deserialize(responseDraft))
     })
 
-    context('when response is part admission - I paid what I believe I owe', () => {
+    context('when response is part admission - I have already paid', () => {
       const responseDraft: ResponseDraft = {
         response: {
           type: ResponseType.PART_ADMISSION
         },
-        rejectPartOfClaim: new RejectPartOfClaim(RejectPartOfClaimOption.PAID_WHAT_BELIEVED_WAS_OWED)
+        partialAdmission: new PartialAdmission().deserialize({
+          alreadyPaid: { option: 'yes' },
+          howMuchHaveYouPaid: { amount: 1, date: { day: 1, mount: 1, year: 1999 }, text: 'aaa' },
```

Comment:
mount -> month?
also in `src/test/features/response/tasks/howMuchHaveYouPaidTask.ts`

Reply:
well spotted